jenkins-slaves/jenkins-slave-helm/Dockerfile

```diff
@@ -2,7 +2,16 @@ FROM quay.io/openshift/origin-jenkins-agent-base:4.4
 
 ARG VERSION=3.2.0 
 ARG YQ_VERSION=3.3.0
+ARG CT_VERSION=3.0.0-rc.1
+ARG GIT_VERSION=2.24.1
+ARG OPENSHIFT_CLIENT_VERSION=4.4.13
```

Comment:
any reason not to use 4.5?

Reply:
Yeah, I wanted to keep the client version consistent with the base image version (4.4)